Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/example/QueueingTask.kt

```diff
@@ -74,4 +77,11 @@ open class QueueingTask @Inject constructor() : Task<DefaultProgress, QueueingTa
         override suspend fun createConfig(): TaskFactory.Config = Config()
         override val taskProvider: () -> Task<DefaultProgress, Result> = { taskByDagger.get() }
     }
+
+    companion object {
+        suspend fun isPreconditionMet(): Boolean{
+            return preconditionIsMet
+        }
+        var preconditionIsMet = true
```

Comment:
I don't quite follow. Can you please elaborate on why this is needed?

Reply:
Queuing task is used for test only (should be move to test directory soon). To test whether task controller skips tasks which preconditions are not yet met, we use this flag to manually indicate in test whether preconditions for this task shall be considered fulfilled